api/v2/app/interfaces/job/manticore/hmi-image-settings.js

```diff
@@ -32,7 +32,7 @@ function generateJobFile (job, body, envs) {
     job.addConstraint({
         LTarget: "${meta.job}",
         Operand: "=",
-        RTarget: "1"
+        RTarget: "true"
```

Comment:
So meta.job is true/false and operand = is not exact equals like ==? The change is that we are comparing true = true instead of true = 1?